.github/workflows/main.yaml

```diff
@@ -0,0 +1,20 @@
+name: Test gha-secret-manager
+on:
+  - push
+
+jobs:
+  secret_manager:
+    runs-on: ubuntu-latest
+    name: Test gha-secret-manager
+    steps:
+      - name: Test gha-secret-manager
+        id: secrets
+        uses: dmsi-io/gha-secret-manager@feature/KNOX-242
```

Comment:
Change tag

Reply:
since this is a new action, there is no `main` tag that's usable so it'll break the check. Since it's just a test action it should be fine to leave it as is since a future developer would have to change it to their feature branch regardless